core/storage/file/gae_models.py

```diff
@@ -157,43 +152,34 @@ class FileSnapshotMetadataModel(base_models.BaseSnapshotMetadataModel):
 
 
 class FileSnapshotContentModel(base_models.BaseSnapshotContentModel):
-    """Class for storing the content of the file snapshots.
-
-    Attributes:
-        content: BlobProperty. The contents of a file snapshot.
-    """
+    """Class for storing the content of the file snapshots (where the content
+    is a JSON blob, i.e. a string)."""
 
     # Overwrite the superclass member to use a BlobProperty for raw strings.
     content = ndb.BlobProperty(indexed=False)
 
 
 class FileModel(base_models.VersionedModel):
-    """File data model, keyed by exploration id and absolute file name.
-
-    Attributes:
-        SNAPSHOT_METADATA_CLASS: FileSnapshotMetadataModel. The class which
-            stores the commit history of the file snapshots.
-        SNAPSHOT_CONTENT_CLASS: FileSnapshotContentModel. The class which
-            stores the content of the file snapshots.
-        content: BlobProperty. The contents of the file.
-    """
+    """File data model, keyed by exploration id and absolute file name."""
+    # The class which stores the commit history of the file snapshots.
     SNAPSHOT_METADATA_CLASS = FileSnapshotMetadataModel
+    # The class which stores the content of the file snapshots.
     SNAPSHOT_CONTENT_CLASS = FileSnapshotContentModel
 
     # The contents of the file.
     content = ndb.BlobProperty(indexed=False)
 
     def _reconstitute(self, snapshot_blob):
-        """Manually overwrite the superclass method. Reconstitutes a FileModel
+        """Overrides the superclass method. Reconstitutes a FileModel
         instance from a snapshot.
 
         Args:
-            snapshot_blob: BlobProperty. A snapshot to which this instance's
-                content field will be set.
+            snapshot_blob: str. A JSON blob to which this instance's content
```

Comment:
str is correct -- I just double-checked by consulting [this reference](https://cloud.google.com/appengine/docs/standard/python/ndb/entity-property-reference#types), too. I don't think "JSON" is part of this, though; it was just an example I gave of another NDB property with a different python value.

So maybe say: "str. A bytestring to which this ..."